paddle/fluid/operators/CMakeLists.txt

```diff
@@ -159,21 +169,30 @@ op_library(create_reader_op DEPS reader)
 
 # Regist multiple Kernel to pybind
 if (WITH_GPU)
-
-op_library(conv_op SRCS conv_op.cc conv_op.cu.cc conv_cudnn_op.cu.cc DEPS
-    vol2col depthwise_conv)
-
-op_library(edit_distance_op SRCS edit_distance_op.cc edit_distance_op.cu DEPS math_function)
-op_library(pool_op SRCS pool_op.cc pool_op.cu.cc pool_cudnn_op.cu.cc DEPS pooling)
-op_library(conv_transpose_op SRCS conv_transpose_op.cc conv_transpose_op.cu.cc
-  conv_transpose_cudnn_op.cu.cc DEPS vol2col)
-file(APPEND ${pybind_file} "USE_OP_DEVICE_KERNEL(conv2d, CUDNN);\n")
-file(APPEND ${pybind_file} "USE_OP_DEVICE_KERNEL(pool2d, CUDNN);\n")
-file(APPEND ${pybind_file} "USE_OP_DEVICE_KERNEL(conv2d_transpose, CUDNN);\n")
+    op_library(edit_distance_op SRCS edit_distance_op.cc edit_distance_op.cu DEPS math_function)
```

Comment:
We will pybind `USE_OP_DEVICE_KERNEL(XXX, CUDNN)` automatically in #8590, in order to make operators/CMakelists.txt much cleaner.

Then, only one sentence `op_library(pool_op DEPS pooling)` will pybind CPU/CUDA/CUDNN/MKLDNN all device kernel.

Thus: 
- how about use `conv_mkldnn_op.cc` likes `conv_cudnn_op.cu.cc` instead of `conv_op.mn.cc`?
- refine these codes after #8590 finished?

Reply:
@luotao1 Do you want my changes to be merged after #8590 is finished?

Reply:
If #8590 doesn't be finished before your small PRs, you can merge your changes at first.

Reply:
#8590 is finished and merged now.